nested/challenges/default.json

```diff
@@ -148,7 +151,8 @@
         {
           "operation": {
             "operation-type": "force-merge",
-            "request-timeout": 7200
+            "request-timeout": 7200{%- if force_merge_max_num_segments is not none %},
```

Comment:
The `{%- endif %}` is missing in this block?

If I run `esrally info --track-path=~/.rally/benchmarks/tracks/default/nested` after I've checked out your PR I get:

```
[ERROR] Cannot info. Could not load '/Users/daniel/.rally/benchmarks/tracks/default/nested/track.json'. The complete track has been written to '/var/folders/pg/gbmwwwms3n50fttdmnqd4sl40000gn/T/tmpxb0h_1qa.json' for diagnosis.
	Unexpected end of template. Jinja was looking for the following tags: 'elif' or 'else' or 'endif'. The innermost block that needs to be closed is 'if'.
```

Reply:
oh, thank you for catching!